4.4/4.4.6/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 4.4.6
+Создать объект со следующей логикой:
+Появляется сообщение: "Введите любой текст (он будет накапливаться)"
+Потом появляется конферм: "Накопленный текст: "[... ... ...]". Хотите еще добавить?"
+Когда пользователь нажмет ОК - начинается всё сначала, появляется первое сообщение.
+Если нажимает ОТМЕНА - всё заканчивается и выводится просто алерт: "Накопленный текст: "[... ... ...]""
+Каждый новый добавленнный текст отделяется от предыдущего с помощью пробела.
+<script>
+	const userTextObj = {
+		text: '',
+		isStart: true,
+		checkInfo() {
+			this.text += prompt('Введите любой текст (он будет накапливаться)');
+			this.text += (this.text) ? ' ' : this.text;
```

Comment:
this condition will always have 'true' and the text expression will always have the space at the end